manim/_config/utils.py

```diff
@@ -1052,8 +1054,9 @@ def format(self, val: str) -> None:
             val,
             [None, "png", "gif", "mp4", "mov", "webm"],
         )
+        self.resolve_movie_file_extension(self.transparent)
```

Comment:
I think this might already be in main (coincidently!) #3839 
This might need to be removed, let's take a look after merging main.